doc/modules/mod_muc_light.md

```diff
@@ -20,6 +20,21 @@ Domain for the MUC Light service to reside under. `@HOST@` is replaced with each
 
 Database backend to use. 
 
+### `modules.mod_muc_light.cache_affs.*`
+  * **Syntax:** TOML section
+  * **Default:** not declared
+  * **Example:** `[modules.mod_muc_light.cache_affs]`
+
+Enables caching affiliations for rooms, this has the advantage that the list of affiliations of a given room is stored locally, instead of being fetched from the DB on each message delivered to a room. On the other hand, in an edge case of a network split when the affiliations of a room are changed, there's a risk of inconsistencies for the cache having values in one node not yet synchronised with the other.
+
+If caching is enabled, it will spawn its own [segmented cache](https://github.com/esl/segmented_cache) cache. To configure the cache parameters, the same config can be accessed within the `cache_affs` key. To see details about the meaning of each flag, see [`mod_cache_users`](./mod_cache_users.md).
```

Comment:
```suggestion
If caching is enabled, it will spawn its own [segmented cache](https://github.com/esl/segmented_cache). To configure the cache parameters, the same config can be stored under the `cache_affs` key. To see details about the meaning of each flag, see [`mod_cache_users`](./mod_cache_users.md).
```